openapi/parser/expand.go

```diff
@@ -137,6 +137,7 @@ func (e *expander) Spec(api *openapi.API) (spec *ogen.Spec, err error) {
 
 func (e *expander) Server(s openapi.Server) (expanded ogen.Server, err error) {
 	expanded.Description = s.Description
+	expanded.URL = s.URL
```

Comment:
Variable `template` already contains stringified server URL template.

This line should be at the end of the function:
```go
expanded.URL = template.String()
```